invenio_users_resources/services/users/tasks.py

```diff
@@ -19,28 +19,38 @@
 
 
 @shared_task(ignore_result=True)
-def reindex_user(user_id):
+def reindex_user(user_ids):
     """Reindex the given user."""
     index = current_users_service.record_cls.index
     if current_users_service.indexer.exists(index):
         try:
-            user_agg = UserAggregate.get_record(user_id)
-            current_users_service.indexer.index(user_agg)
+            user_agg = {
+                user_id: UserAggregate.get_record(user_id) for user_id in user_ids
+            }
+            current_users_service.indexer.bulk_index(user_ids)
+
             # trigger reindexing of related records
             send_change_notifications(
```

Comment:
This is actually a Celery task from what I see, so we should be explicit in our call, i.e.:

```python
task = send_change_notifications.si("users", [...])

# Asynchronsouly
task.delay()

# Synchronously
task.apply(throw=True)
```